modules/ossm-rn-known-issues.adoc

```diff
@@ -23,7 +23,7 @@ These limitations exist in {ProductName}:
 
 * The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
 
-* WebAssembly is unsupported on IBM Z and IBM Power Systems.
+* WebAssembly is unsupported on IBM Z Systems.
```

Comment:
@rh-tokeefe Can you please also remove "Systems".  
**IBM Z** is a brand name and **IBM Power Systems** is as well.